client/allocrunner/csi_hook.go

```diff
@@ -94,24 +106,28 @@ func (c *csiHook) Postrun() error {
 	return result.ErrorOrNil()
 }
 
+type volumeAndRequest struct {
+	volume  *structs.CSIVolume
+	request *structs.VolumeRequest
+}
+
 // csiVolumesFromAlloc finds all the CSI Volume requests from the allocation's
 // task group and then fetches them from the Nomad Server, before returning
 // them in the form of map[RequestedAlias]*structs.CSIVolume.
```

Comment:
Let's update the docstring to match the new return value.

Reply:
It's updated in the downstream PR of this right now :sweat:

Reply:
Just saw that 👍